RobotFramework_TestsuitesManagement/Config/CConfig.py

```diff
@@ -339,8 +339,10 @@ def loadCfg(self):
                     raise Exception(f"The parameter {error.message}, but it's missing in JSON configuration file.")
                 else:
                     errParam = error.path.pop()
-                    logger.error(f"Parameter '{errParam}' in JSON configuration file is not allowed due to: {error.message}")
-                    raise Exception(f"Parameter '{errParam}' in JSON configuration file is not allowed due to: {error.message}")
+                    logger.error(f"The value sets for the parameter '{errParam}' in JSON configuration file is \
```

Comment:
If a string ends with \ (to indicate that the string will be continued in the next line) then this following line must not be indented! Otherwise you will have the horizontal space of the indentation as part of the resulting output string in console and log file.